core/src/js/components/main-nav/main-nav.js

```diff
@@ -6,22 +6,23 @@ document.addEventListener('DOMContentLoaded', event => {
 
   // The css class that this following behaviour is applied to.
   const navClass = 'su-main-nav';
-
-  // Variables.
-
   // All main navs.
   const navs = document.querySelectorAll('.' + navClass);
 
-  // ---------------------------------------------------------------------------
-  // Execution Code.
-  // ---------------------------------------------------------------------------
-
   // Process each nav.
   let firstZindex;
   navs.forEach((nav, index) => {
+    // remove the class that formats the nav for browsers with javascript disabled
+    nav.classList.remove('no-js');
```

Comment:
This is happening in the template file. Can we remove that javascript code from there now?

Reply:
I thought I still saw a brief FOUC when I removed it from the template file, but it seems ok now. I'm happy to remove it.